sql/load_demo_data.sql

```diff
@@ -5388,7 +5561,9 @@ CREATE TABLE `oh_permissions` (
   `P_CREATED_DATE` datetime DEFAULT NULL,
   `P_LAST_MODIFIED_BY` varchar(50) DEFAULT NULL,
   `P_LAST_MODIFIED_DATE` datetime DEFAULT NULL,
-  PRIMARY KEY (`P_ID_A`)
+  PRIMARY KEY (`P_ID_A`),
+  KEY `FK_PERMISSIONS_CREATED_BY_IDX` (`P_CREATED_BY`),
+  KEY `FK_PERMISSIONS_LAST_MODIFIED_BY_IDX` (`P_LAST_MODIFIED_BY`)
```

Comment:
Missing constraints and wrong Engine:
<pre>
 CONSTRAINT `FK_PERMISSIONS_CREATED_BY` FOREIGN KEY (`P_CREATED_BY`) REFERENCES `oh_user` (`US_ID_A`) ON DELETE NO ACTION ON UPDATE NO ACTION,
  CONSTRAINT `FK_PERMISSIONS_LAST_MODIFIED_BY` FOREIGN KEY (`P_LAST_MODIFIED_BY`) REFERENCES `oh_user` (`US_ID_A`) ON DELETE NO ACTION ON UPDATE NO ACTION
</pre>

Engine in demo data is MyISAM and in the multiple step version it is InnoDB

Reply:
Same as for other MyISAM tables.
Added `ALTER TABLE OH_PERMISSIONS ENGINE = INNODB, CONVERT TO CHARACTER SET utf8;` to step_95.
Thanks!

Reply:
Once again:
Load demo data missing FK:
![image](https://github.com/informatici/openhospital-core/assets/1105445/60659de9-b716-4843-87eb-2b7eea145d0f)

Load step 04:
![image](https://github.com/informatici/openhospital-core/assets/1105445/3aa7fe8b-254f-4056-8963-7c10e1b93cfd)

Reply:
could you past here the missing lines please?

Reply:
<pre>
  CONSTRAINT `FK_PERMISSIONS_CREATED_BY` FOREIGN KEY (`P_CREATED_BY`) REFERENCES `oh_user` (`US_ID_A`) ON DELETE NO ACTION ON UPDATE NO ACTION,
  CONSTRAINT `FK_PERMISSIONS_LAST_MODIFIED_BY` FOREIGN KEY (`P_LAST_MODIFIED_BY`) REFERENCES `oh_user` (`US_ID_A`) ON DELETE NO ACTION ON UPDATE NO ACTION
</pre>